.github/workflows/docker.yml

```diff
@@ -11,13 +11,9 @@ on:
 
 jobs:
   docker-amd64:
-    runs-on: bare-metal
+    runs-on: ubuntu-latest
     name: Build DockerHub AMD64 Container
     steps:
-      - name: Install jq
-        run: sudo apt-get install jq -y
-      - name: Prune stale references
-        run: git remote prune origin
```

Comment:
no longer needed since we're sandboxed now by virtue of running on not-bare-metal